dart/test/test_utils.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:convert';
 
 import 'package:http/http.dart';
 import 'package:http/testing.dart';
-import 'package:sentry/sentry.dart';
+import 'package:sentry/sentry.dart' hide Request;
```

Comment:
why this if its just a test?

Reply:
If we expose the Request class, we need to avoid the class name conflict with`http.Request`.